edit/workload/index.vue

```diff
@@ -875,6 +891,15 @@ export default {
         </Tab>
         <Tab :label="t('workload.container.titles.securityContext')" name="securityContext">
           <Security v-model="container.securityContext" :mode="mode" />
+          <div class="spacer"></div>
+          <div>
+            <h3>{{ t('workload.container.security.podFsGroup') }}</h3>
+            <div class="row">
+              <div class="col span-6">
+                <LabeledInput v-model.number="podFsGroup" :mode="mode" :label="t('workload.container.security.fsGroup')" />
```

Comment:
Looks like `v-model.number` doesn't quite work as expected on `LabeledInput` (it will accept non-numerical values). Think `type="number"` is also needed.